docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+# Lifecycle Manager - Operations
+
+## Metrics
+
+Lifecycle Manager metrics are exposed over port `8080` through the metrics endpoint `/metrics`.
+
+The following metrics are exposed:
+
+| Metric Name                              | Metric Type    | Description                                                                                  |
+|------------------------------------------|----------------|----------------------------------------------------------------------------------------------|
+| `lifecycle_mgr_requeue_reason_total`     | Counter Vector | Indicates the requeue reason of a reconciler.                                                |
+| `lifecycle_mgr_kyma_state`               | Gauge Vector   | Indicates the state of a Kyma CR.                                                            |
+| `lifecycle_mgr_module_state`             | Gauge Vector   | Indicates the state of a module added to a Kyma CR.                                        |
+| `lifecycle_mgr_mandatory_modules`        | Gauge          | Indicates the number of mandatory ModuleTemplate CRs.                                           |
+| `lifecycle_mgr_mandatory_module_state`   | Gauge Vector   | Indicates the state of a mandatory module added to a Kyma CR.                              |
+| `reconcile_duration_seconds`             | Gauge Vector   | Indicates the duration of a Manifest CR reconciliation in seconds.                              |
+| `lifecycle_mgr_purgectrl_time`           | Gauge          | Indicates the average duration of purge reconciliation.                                      |
+| `lifecycle_mgr_purgectrl_requests_total` | Counter        | Indicates the total number of purges.                                                        |
+| `lifecycle_mgr_purgectrl_error`          | Gauge Vector   | Indicates the errors produced by the purge.                                                      |
+| `lifecycle_mgr_self_signed_cert_not_renew` | Gauge Vector  | Indicates that the self-signed Certificate of a Kyma CR is not renewed yet.                 |
```

Comment:
Where can we read more about this mechanics? Just by this description this is not clear for an outsider, whether this is an error (not renewed yet, but should have been already), or just a FYI-style indicator.